lib/syslog.ml

```diff
@@ -54,7 +54,6 @@ let level_of_string s =
 	| "emergency"        -> Emerg
 	| "alert"            -> Alert
 	| "critical"         -> Crit
-	| "debug"            -> Debug; 
```

Comment:
Why can this line be removed? The problem I see here is the semicolon, not the rest of the line.